src/app/categories/[categoryId]/pairwise-ranking/comment/page.tsx

```diff
@@ -58,6 +59,8 @@ const CategoryRankingComment = () => {
 	const ranking = rankingRes?.data;
 
 	const attest = async () => {
+		posthog.capture("Attested",{ attestedCategory:category?.data.collection?.name })
```

Comment:
Can we do it at the end of this function as we only care about successful attestations?